src/components/Search/Search.tsx

```diff
@@ -10,10 +10,22 @@ import Meta from '../Meta'
 import { canSearchForProperty } from '@/utils/userPermissions'
 import { PropertyListItem } from '@/models/propertyListItem'
 
-const Search = ({ query }) => {
+interface SearchProps {
+  query?: {
+    searchText?: string
+    pageNumber?: string
```

Comment:
Guess it's the same issue with the pageNumber:
```ts
...(pageNumber && { pageNumber: parseInt(pageNumber) }),
```

Reply:
The repairs API seems to treat this as an integer [[link to model](https://github.com/LBHackney-IT/repairs-api-dotnet/blob/423055144f1f7df6a8942c94f3d2c0906c0bca9b/RepairsApi/V2/Boundary/Request/SearchForPropertiesRequest.cs#L18)].